### ISSUE TITLE
use kebab-case for inputs

### ISSUE THREAD

**Issue by mattwynne** [cucumber/action-publish-rubygem#2]
For consistency with other github actions.

Rename  e.g. `working_directory` to `working-directory`

**Comment by luke-hill** [comment]
@mattwynne is this still relevant? the `with` param on github actions is from their API, so I think it needs to be snake cased?

**Comment by mattwynne** [comment]
Yeah, it's still relevant. It's minor though. The work would be in going over all the places where we use it (e.g. [here](https://github.com/cucumber/gherkin/blob/main/.github/workflows/release-rubygem.yml#L20-L21)) to change the way we're using it.

**Comment by mpkorstanje** [comment]
Done, coincidentally, in https://github.com/cucumber/action-publish-rubygem/pull/25.
